src/Periphery/ServiceFeeCollector.sol

```diff
@@ -68,6 +69,10 @@ contract ServiceFeeCollector is TransferrableOwnership {
             receiver,
             msg.value
         );
```

Comment:
Why do we need that function still? We collect gas fees in the LiFuelFeeCollector.
Also the event emits `msg.value` instead of the `feeAmount`.